test/functional/apps/discover/group1/_discover_accessibility.ts

```diff
@@ -75,7 +80,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await focusAndPressButton('discoverAlertsButton');
         expect(await hasFocus('discoverAlertsButton')).to.be(false);
         await focusAndPressButton('discoverCreateAlertButton');
-        expect(await testSubjects.exists('addRuleFlyoutTitle')).to.be(true);
+        // Increase timeout. The rule flyout has to load asynchronously when the user clicks the create rule button,
+        // so a higher timeout here reduces flakiness in the test
+        expect(await testSubjects.exists('addRuleFlyoutTitle', { timeout: 20000 })).to.be(true);
```

Comment:
We can try to update `test/functional/apps/discover/group1/config.ts` with the following:

```
export default async function ({ readConfigFile }: FtrConfigProviderContext) {
  const functionalConfig = await readConfigFile(require.resolve('../../../config.base.js'));

  return {
    ...functionalConfig.getAll(),
    kbnTestServer: {
      ...functionalConfig.get('kbnTestServer'),
      serverArgs: [
        ...functionalConfig.get('kbnTestServer.serverArgs'),
        '--xpack.encryptedSavedObjects.encryptionKey="wuGNaIhoMpk5sO4UBxgr3NyW1sFcLgIf"', // it's required for alerts plugin to work
      ],
    },
    testFiles: [require.resolve('.')],
  };
}
```